diff --git a/src/StaticATokenLM.sol b/src/StaticATokenLM.sol
index 1416b85..4663cb9 100644
--- a/src/StaticATokenLM.sol
+++ b/src/StaticATokenLM.sol
@@ -661,7 +661,7 @@ contract StaticATokenLM is
   }
 
   function _convertToShares(uint256 amount, Rounding rounding) internal view returns (uint256) {
-    if (rounding == Rounding.UP) return amount.rayDivRoundUp(rate());
+    /// FunctionCallMutation: if (rounding == Rounding.UP) return amount.rayDivRoundUp(rate());
     return amount.rayDivRoundDown(rate());
   }
 
