diff --git a/src/StaticATokenLM.sol b/src/StaticATokenLM.sol
index 1416b85..a1b78da 100644
--- a/src/StaticATokenLM.sol
+++ b/src/StaticATokenLM.sol
@@ -373,10 +373,8 @@ contract StaticATokenLM is
     }
 
     // otherwise users can withdraw up to the available amount
-    uint256 underlyingTokenBalanceInShares = _convertToShares(
-      IERC20(cachedATokenUnderlying).balanceOf(reserveData.aTokenAddress),
-      Rounding.DOWN
-    );
+    /// FunctionCallMutation of: uint256 underlyingTokenBalanceInShares = _convertToShares(
+    uint256 underlyingTokenBalanceInShares = IERC20(cachedATokenUnderlying).balanceOf(reserveData.aTokenAddress);
     uint256 cachedUserBalance = balanceOf[owner];
     return
       underlyingTokenBalanceInShares >= cachedUserBalance
